packages/react-params/src/index.tsx

```diff
@@ -80,6 +82,14 @@ class Params extends React.PureComponent<Props, State> {
       return null;
     }
 
+    if (values && params) {
+      params.map(({ type }: ParamDef, index: number) => {
+        if (type.type?.includes('AssetId')) {
+          this.state.assetIdContext = values[index].value.toString();
```

Comment:
when a dynamic extrinsic or query component renders we track any asset ID fields and pass that to balance components later on

Reply:
do we need to do 
`
this.setState({ assetIdContext: values[index].value.toString() });
`
to set the state